### PR TITLE
[improve][doc] improve `pulsar-admin namespaces set-backlog-quota`

### PR DESCRIPTION
**Pull request by labuladong**
The document update of https://github.com/apache/pulsar/pull/19300

Please review this pr only if https://github.com/apache/pulsar/pull/19300 has been merged.

### CODE REVIEW

**Comment by labuladong** [comment]
This pr is ready to merge @Anonymitaet @momo-jun

File: docs/reference-pulsar-admin.md [hunk 1]
```diff
@@ -1240,26 +1240,29 @@ pulsar-admin namespaces set-backlog-quota tenant/namespace options
...
-|`-p`, `--policy`|The retention policy to enforce when the limit is reached. The valid options are: `producer_request_hold`, `producer_exception` or `consumer_backlog_eviction`|
-|`-t`, `--type`|Backlog quota type to set. The valid options are: `destination_storage`, `message_age` |destination_storage|
+| Flag                 | Description                                                                                                                                                                                          | Default             |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| `-t`, `--type`       | Backlog quota type to set. The valid options are: `destination_storage`, `message_age`                                                                                                               | destination_storage |
```
Comment (momo-jun) [inline docs/reference-pulsar-admin.md:1245]:
```suggestion
| `-t`, `--type`       | The backlog quota type. The valid options are: `destination_storage` and `message_age`.                                                                                                               | destination_storage |
```

File: docs/reference-pulsar-admin.md [hunk 2]
```diff
@@ -1240,26 +1240,29 @@ pulsar-admin namespaces set-backlog-quota tenant/namespace options
...
+| `-l`, `--limit`      | If you set backlog quota type to `destination_storage`, this parameter can set the backlog size limit (for example `10M` or `16G`), non-positive number for disabling size limit.                    ||
```
Comment (momo-jun) [inline docs/reference-pulsar-admin.md:1246]:
```suggestion
| `-l`, `--limit`      | If the backlog quota type is set to `destination_storage`, you can use this parameter to set the backlog size limit (for example, `10M` or `16G`). To disable the size limit, use a non-positive number.                    ||
```

File: docs/reference-pulsar-admin.md [hunk 3]
```diff
@@ -1240,26 +1240,29 @@ pulsar-admin namespaces set-backlog-quota tenant/namespace options
...
+| `-lt`, `--limitTime` | If you set backlog quota type to `message_age`, this parameter can set the time limit in second (or minutes, hours, days, weeks eg: 100m, 3h, 2d, 5w), non-positive number for disabling time limit. ||
+| `-p`, `--policy`     | The retention policy to enforce when the limit is reached. The valid options are: `producer_request_hold`, `producer_exception` or `consumer_backlog_eviction`                                       |
+
+If you want set both time limit and size limit, you can set them separately.
```
Comment (momo-jun) [inline docs/reference-pulsar-admin.md:1250]:
```suggestion
If you want to set both the time limit and the size limit, you need to set them separately.
```

File: docs/reference-pulsar-admin.md [hunk 4]
```diff
@@ -1240,26 +1240,29 @@ pulsar-admin namespaces set-backlog-quota tenant/namespace options
...
+| Flag                 | Description                                                                                                                                                                                          | Default             |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| `-t`, `--type`       | Backlog quota type to set. The valid options are: `destination_storage`, `message_age`                                                                                                               | destination_storage |
+| `-l`, `--limit`      | If you set backlog quota type to `destination_storage`, this parameter can set the backlog size limit (for example `10M` or `16G`), non-positive number for disabling size limit.                    ||
+| `-lt`, `--limitTime` | If you set backlog quota type to `message_age`, this parameter can set the time limit in second (or minutes, hours, days, weeks eg: 100m, 3h, 2d, 5w), non-positive number for disabling time limit. ||
```
Comment (momo-jun) [inline docs/reference-pulsar-admin.md:1247]:
```suggestion
| `-lt`, `--limitTime` | If the backlog quota type is set to `message_age`, you can use this parameter to set the time limit in seconds (or minutes, hours, days, weeks, e.g., 100m, 3h, 2d, 5w). To disable the time limit, use a non-positive number. ||
```

File: docs/reference-pulsar-admin.md [hunk 5]
```diff
@@ -1240,26 +1240,29 @@ pulsar-admin namespaces set-backlog-quota tenant/namespace options
...
+| `-p`, `--policy`     | The retention policy to enforce when the limit is reached. The valid options are: `producer_request_hold`, `producer_exception` or `consumer_backlog_eviction`                                       |
```
Comment (momo-jun) [inline docs/reference-pulsar-admin.md:1248]:
```suggestion
| `-p`, `--policy`     | The retention policy to enforce when the limit is reached. The valid options are: `producer_request_hold`, `producer_exception`, and `consumer_backlog_eviction`.                                       |
```

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**